include/igl/line_field_mismatch.cpp

```diff
@@ -46,8 +46,8 @@ class MissMatchCalculatorLine
 private:
 
     //compute the mismatch between 2 faces
-    inline int MissMatchByLine(const int f0,
-                               const int f1)
+    inline int MisMatchByLine(const int f0,
```

Comment:
Should be `MismatchByLine` now, or maybe even `mismatchByLine` (also camelCase).